setup.cfg

```diff
@@ -0,0 +1,82 @@
+[aliases]
+dists = clean --all sdist bdist_wheel
+
+[metadata]
+name = ansible-compat
+url = https://github.com/ansible-community/ansible-compat
+project_urls =
+  Bug Tracker = https://github.com/ansible-community/ansible-compat/issues
+  CI: GitHub = https://github.com/ansible-community/ansible-compat/actions?query=workflow:gh+branch:master+event:push
+  Code of Conduct = https://docs.ansible.com/ansible/latest/community/code_of_conduct.html
+  Source Code = https://github.com/ansible-community/ansible-compat
+description = Ansible compatibility goodies
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Sorin Sbarnea
+author_email = ssbarnea@redhat.com
+maintainer = Sorin Sbarnea
+maintainer_email = ssbarnea@redhat.com
```

Comment:
I normally set the maintainer field if it's different from the author. Not sure if there's any policy wrt tho